src/app/about/about.component.html

```diff
@@ -36,29 +36,29 @@ <h5>
     </div>
 
     <div fxLayout="column" class="container" fxLayoutAlign="space-between center" fxLayoutGap="1rem">
-      <h3>Dockstore follows the FAIR Guiding Principles</h3>
+      <h2>Dockstore follows the FAIR Guiding Principles</h2>
       <span
         >Our
         <a
           target="_blank"
           rel="noopener noreferrer"
-          [href]="Dockstore.DOCUMENTATION_URL + 'advanced-topics/best-practices/best-practices-secure-fair-workflows.html'"
```

Comment:
Assuming this is intended

Reply:
Hard-coding this does seem a bit odd

Reply:
In discussion, oddly https://docs.dockstore.org/en/develop/advanced-topics/best-practices/best-practices-secure-fair-workflows.html exists but https://docs.dockstore.org/en/latest/advanced-topics/best-practices/best-practices-secure-fair-workflows.html does not

Reply:
Probably due to https://ucsc-cgl.atlassian.net/browse/SEAB-2737